### ISSUE TITLE
Model performance is very poor aft SFT

### ISSUE THREAD

**Issue by backkon** [om-ai-lab/VLM-R1#96]
Hello, thank you for your open source to promote the research of multimodal R1. I found a problem in the experiment and would like to discuss it with you. I found that when I performed SFT on the Refcoco/+/g dataset, the score on the evaluation set became lower and lower as the training progressed. Do you have the same problem? This does not seem to be in line with common sense. Even if the SFT effect is not as good as RL, it should be better than the baseline.

**Comment by BrightHai** [comment]
Maybe overfit on the training data, try set the lora_r=8/16, lora_alpha=32, learning rate=1e-5
